python/sports-league-scheduling/src/sports_league_scheduling/constraints.py

```diff
@@ -81,7 +81,7 @@ def start_to_away_hop(constraint_factory: ConstraintFactory) -> Constraint:
             .for_each(Match)
             .if_not_exists(Round, Joiners.equal(lambda match: match.round.index - 1,
                                                 lambda match_round: match_round.index))
-            .penalize(HardSoftScore.ONE_SOFT, away_home_match_distance_lambda)
+            .penalize(HardSoftScore.ONE_SOFT, lambda match: (match.away_team.get_distance(match.home_team)))
```

Comment:
Parenthesis uneccessary

```suggestion
            .penalize(HardSoftScore.ONE_SOFT, lambda match: match.away_team.get_distance(match.home_team))
```